docs/mp/telemetry.adoc

```diff
@@ -0,0 +1,408 @@
+
+Please consult with the links above for all configurations properties usage.
+
+The property should be declared in `microprofile-config.properties` file in order to be processed correctly.
+
+
+=== OpenTelemetry Java Agent
+
+The OpenTelemetry Java Agent may influence the work of MicroProfile Telemetry, on how the objects are created and configured. Helidon will do "best effort" detect the use of the agent. But, if there is a decision to run the Helidon app with the agent, a configuration property should be set:
+
+`otel.agent.present=true`
+
+This way, Helidon will explicitly get all the configuration and objects from the Agent, thus allowing correct Span hierarchy settings.
+
+== Examples
+
+This guide showcases how to incorporate MicroProfile Telemetry into Helidon and provides illustrations of how to view traces. Jaeger is employed in all the examples, and the Jaeger UI is used to view the traces.
+
+=== Setup Jaeger
+
+For the examples Jaeger will be used for gathering of the tracing information.
+
+.Run Jaeger in a docker container.
+[source, bash]
+----
+docker run -d --name jaeger \
+  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
+  -e COLLECTOR_OTLP_ENABLED=true \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 4317:4317 \
+  -p 4318:4318 \
+  -p 14250:14250 \
+  -p 14268:14268 \
+  -p 14269:14269 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:1.41
+----
+
+All the tracing information gathered from the examples runs is accessible from the browser in the Jaeger UI under link:http://localhost:16686/[]
+
+=== Enable MicroProfile Telemetry in Helidon Application
+
+Together with Helidon Telemetry dependency, an OpenTelemetry Exporter dependency should be added to project's pom.xml file.
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.telemetry</groupId>
+    <artifactId>helidon-microprofile-telemetry</artifactId> <1>
+</dependency>
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>  <2>
+</dependency>
+----
+<1> Helidon Telemetry dependency.
+<2> OpenTelemetry Jaeger exporter.
+
+Add these lines to `META-INF/microprofile-config.properties`:
+
+.MicroProfile Telemetry properties
+[source,properties]
+----
+otel.sdk.disabled=false     <1>
+otel.traces.exporter=jaeger <2>
+otel.exporter.name=greeting-service <3>
+----
+<1> Enable MicroProfile Telemetry.
+<2> Set exporter to Jaeger.
+<3> Name of our service.
+
+Here we enable MicroProfile Telemetry, set tracer to "jaeger" and give a name, which will be used to identify our service in the tracer.
+
+[NOTE]
+====
+for our example we use Jaeger managing the tracing data. If you prefer to use Zipkin, please set `otel.traces.exporter` property to "zipkin". To find more information on how to use Zipkin, please proceed to link:https://zipkin.io/[]. Also a corresponding Maven dependency for the exporter should be added:
```

Comment:
For this example, you will use Jaeger to manage data tracing. 


For more information using about Zipkin, see <link>

Reply:
Fixed.